src/components/EditAccount.jsx

```diff
@@ -139,5 +150,11 @@ class EditAccount extends Component {
   }
 }
 
+const mapDispatchToProps = { editAccount:editAccount }
+const mapStateToProps = (state) => {
+  return {
+     Account: state.accounts
```

Comment:
since this prop holds multiple accounts, do you think a better name would be "accounts" instead of "Account"?